clang/lib/FrontendTool/ExecuteCompilerInvocation.cpp

```diff
@@ -53,8 +66,13 @@ CreateFrontendBaseAction(CompilerInstance &CI) {
   case DumpTokens:             return std::make_unique<DumpTokensAction>();
   case EmitAssembly:           return std::make_unique<EmitAssemblyAction>();
   case EmitBC:                 return std::make_unique<EmitBCAction>();
+#if CLANG_ENABLE_CIR
+  case EmitCIR:
```

Comment:
So I'd like 1 of 2 things to happen here: 
1- move the 'case' outside of the #if/#else
2- OR put both 'case' and 'make_unique' (and perhaps llvm_unreachable)  on a single line with the spacing to match the rest of the file.